packages/kit/test/apps/basics/test/test.js

```diff
@@ -1283,6 +1283,18 @@ test.describe('Redirects', () => {
 		);
 	});
 
+	test('errors on invalid 3xx status', async ({ baseURL, page, clicknav }) => {
```

Comment:
doing an e2e test for this is kind of overkill, a unittest would be enough. But since that unit test tests the same logic we have inside the function, it's kind of pointless to have a unit test anyway in my opion 😄

Reply:
I just looked for `redirect` in the tests folder and copied an existing one. But I can get rid of it